src/SFA.DAS.EmployerFinance.Web/SFA.DAS.EmployerFinance.Web.csproj

```diff
@@ -10,6 +10,8 @@
         <PackageReference Include="BuildBundlerMinifier" Version="2.8.391" />
         <PackageReference Include="Microsoft.AspNetCore.App" />
         <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
+        <PackageReference Include="NLog" Version="4.5.11" />
+        <PackageReference Include="NLog.Web.AspNetCore" Version="4.8.0" />
         <PackageReference Include="StructureMap" Version="4.7.0" />
```

Comment:
The `SFA.DAS.NLog.Targets.Redis` will also need to be referenced.

Reply:
done